test/local/routes/sign.js

```diff
@@ -12,11 +12,12 @@ const uuid = require('uuid')
 const getRoute = require('../../routes_helpers').getRoute
 const mocks = require('../../mocks')
 const P = require(`${ROOT_DIR}/lib/promise`)
+const error = require(`${ROOT_DIR}/lib/error`)
 
 describe('/certificate/sign', () => {
   let db, deviceId, mockDevices, mockLog, sessionToken, mockRequest
 
-  before(() => {
+  beforeEach(() => {
```

Comment:
I had to switch to a `beforeEach` here because I completely replace the shared `mockDevices` object in my new test.  But I think `beforeEach` is nicer than a bunch of `finally` clauses anyway.